### PR TITLE
fix: Adjust Infos UI

### PR DESCRIPTION
**Pull request by y-lohse**
This PR makes some small UI tweaks for the Infos component, as seen with @joel-costa .

My preview branch is busy with #1347 but I think we can do without it here, it's just changing some spacing values.

### CODE REVIEW

File: react/InfosCarrousel/styles.styl [hunk 1]
```diff
@@ -12,16 +13,20 @@
 
 .InfosCarrousel-navigation
     position absolute
-    bottom 0
-    right 0
+    bottom 10px
+    right 8px
```
Comment (drazik) [inline react/InfosCarrousel/styles.styl]: This can be replaced by `spacing_values.xs`

File: react/InfosCarrousel/styles.styl [hunk 2]
```diff
@@ -12,16 +13,20 @@
...
     align-items center
 
+    +mobile()
+        bottom 4px
+        right 0
+
 .InfosCarrousel-arrow
     background none
     border 0
     min-width 32px
```
Comment (drazik) [inline react/InfosCarrousel/styles.styl]: This can be replaced by `spacing_values.xl`

File: react/InfosCarrousel/styles.styl [hunk 3]
```diff
@@ -12,16 +13,20 @@
...
-    padding 8px 16px
+    padding 16px
```
Comment (drazik) [inline react/InfosCarrousel/styles.styl]: This can be replaced by `spacing_values.m`

File: react/InfosCarrousel/styles.styl [hunk 4]
```diff
@@ -12,16 +13,20 @@
 
 .InfosCarrousel-navigation
     position absolute
-    bottom 0
-    right 0
+    bottom 10px
```
Comment (drazik) [inline react/InfosCarrousel/styles.styl]: Can we use `rem(10)` here?

File: react/InfosCarrousel/styles.styl [hunk 5]
```diff
@@ -12,16 +13,20 @@
...
+    right 8px
     display flex
     align-items center
 
+    +mobile()
+        bottom 4px
```
Comment (drazik) [inline react/InfosCarrousel/styles.styl]: Can we use `rem(4)` here?

File: react/InfosCarrousel/styles.styl (hunk 2 above)
Comment (ptbrowne) [reply react/InfosCarrousel/styles.styl]: Are we sure that a min-width should be implemented with a spacing value ?

**Pull request merged**